hieradata/nodes/lethe.yaml

```diff
@@ -0,0 +1,4 @@
+classes:
+  - ocf_postgres
+
+ocf::ssl::default::owner: root
```

Comment:
If this is a hard requirement of postgres, it would be preferable for the `ocf_postgres` class to set this setting. Remind me, is this class being included from elsewhere?

Reply:
the reason I had to put it in hiera is because of this: https://puppet.com/docs/puppet/5.3/lang_classes.html#include-like-behavior

> Include-like behavior relies on external data and defaults for class parameter values, which allows the external data source to act like cascading configuration files for all of your classes.

otherwise I'd need to do 
```puppet
ocf::ssl::default {
  owner => 'root',
}
```

I'm not attached to either approach